src/AppKit/Enums.cs

```diff
@@ -2154,10 +2154,191 @@ public enum NSPopUpArrowPosition : ulong {
 		Bottom
 	}
 
-	public static class NSFileTypeForHFSTypeCode {
-		public static readonly string ComputerIcon = "root";
-		public static readonly string DesktopIcon = "desk";
-		public static readonly string FinderIcon = "FNDR";
+	// FileType 4cc values to use with NSFileTypeForHFSTypeCode.
+	public enum HFSTypeCode: uint
+	{
+		/* Generic Finder icons */
+		ClipboardIcon                 = 0x434C4950,   //'CLIP'
+		ClippingUnknownTypeIcon       = 0x636C7075,   //'clpu'
+		ClippingPictureTypeIcon       = 0x636C7070,   //'clpp'
+		ClippingTextTypeIcon          = 0x636C7074,   //'clpt'
+		ClippingSoundTypeIcon         = 0x636C7073,   //'clps'
+		DesktopIcon                   = 0x6465736B,   //'desk'
+		FinderIcon                    = 0x464E4452,   //'FNDR'
+		ComputerIcon                  = 0x726F6F74,   //'root'
+		FontSuitcaseIcon              = 0x4646494C,   //'FFIL'
+		FullTrashIcon                 = 0x66747268,   //'ftrh'
+		GenericApplicationIcon        = 0x4150504C,   //'APPL'
+		GenericCDROMIcon              = 0x63646472,   //'cddr'
```

Comment:
`Cdrom`
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers